oak_functions/lookup/src/lib.rs

```diff
@@ -153,10 +153,8 @@ pub struct LookupDataManager<L: OakLogger + Clone> {
 }
 
 #[derive(Debug, PartialEq)]
-pub enum UpdateStatus {
-    Extended,
-    Finished,
-    Aborted,
+pub struct NextDataStatus {
+    pub ok: bool,
```

Comment:
Could you clarify the semantics of this? Is it just a wrapper around a bool? If so, maybe it could be an actual stand-alone enum with SUCCESS / FAILURE, or a Result<(), ()>?